docs/Displaying a User Interface/Alerts and Subtle Alerts/index.md

```diff
@@ -251,8 +251,6 @@ alertView.setAudio(alertAudioData);
 ```
 !@
 
-You can also play a combination of audio files and text-to-speech strings. The audio will be played in the order you add them to the @![android,javaSE,javaEE, javascript]`AlertAudioData`!@@![iOS]`SDLAlertAudioData`!@ object.
-
```

Comment:
Why was this removed?

Reply:
I have no idea.  This was not an intentional change.  I will investigate.

Reply:
This looks like a copy/paste error.  It looks like when I added some text, there was something highlighted already that got replaced.